solver/src/main.rs

```diff
@@ -338,13 +354,27 @@ async fn main() {
         .await
         .map_err(|err| tracing::error!("Couldn't fetch market makable token list: {}", err))
         .ok();
+    let solution_submitter = SolutionSubmitter {
+        web3: web3.clone(),
+        contract: settlement_contract.clone(),
+        account,
+        gas_price_estimator: gas_price_estimator.clone(),
+        target_confirm_time: args.target_confirm_time,
```

Comment:
Are gas_price_estimator and target_confirm_time used in archer? I thought they provide an endpoint where they give us "tip" estimates?

Reply:
They are used because we use the same gas estimator. I guess I didn't document this anywhere but when I tested their endpoint I didn't like that it was a private endpoint without documentation and that their estimates seemed way too high. According to the flash bots documentation (and this makes sense to me too) you just need to pay miners enough that they prefer your transaction over the currently lowest gas price transaction in the mining block. This is the same as for public mempool transactions so public gas price estimators should be fine.